rotkehlchen/data/importer.py

```diff
@@ -182,3 +186,183 @@ def import_cointracking_csv(self, filepath: Path) -> None:
                     continue
 
         return None
+
+    def _consume_cryptocom_entry(self, csv_row: Dict[str, Any]) -> None:
+        """Consumes a cryptocom entry row from the CSV and adds it into the database
+        Can raise:
+            - DeserializationError if something is wrong with the format of the expected values
+            - UnsupportedCryptocomEntry if importing of this entry is not supported.
+            - IndexError if the CSV file is corrupt
```

Comment:
Why `IndexError`? From what I see since each row is a dictionary if it has a missing key it will throw a `KeyError`. For example if `csv_row` does not have a key for `'Transaction Kind'` it will raise `KeyError`

Reply:
It was copied from `_consume_cointracking_entry` 🙈
So I can change that for both methods here?

Reply:
I guess then both may be possible. You would have to check.

So it's like:

1) IndexError if the number of rows is not the same as expected?
2) KeyError if a key is missing. That would happen if in the first line, the header line, one column is named differently than expected.